ansible/roles/kubernetes/tasks/secrets.yml

```diff
@@ -80,16 +80,39 @@
     - restart daemons
 
 - include: gen_tokens.yml
-  when: inventory_hostname == groups['masters'][0]
```

Comment:
Why you delete this condition and add the same condition to every task in the `gen_tokens.yml` ?  
I can not see any difference of what you have changed...